src/content/docs/en/reference/api-reference.mdx

```diff
@@ -808,6 +921,88 @@ const { page } = Astro.props;
 
 Pagination will pass a `page` prop to every rendered page that represents a single page of data in the paginated collection. This includes the data that you've paginated (`page.data`) as well as metadata for the page (`page.url`, `page.start`, `page.end`, `page.total`, etc). This metadata is useful for things like a "Next Page" button or a "Showing 1-10 of 100" message.
 
+##### `page.data`
+
+<p>
+**Type:** `Array`
+</p>
+
+Array of data returned from `data()` for the current page.
+
+##### `page.start`
+
+<p>
+**Type:** `number`
+</p>
+
+Index of first item on current page, starting at `0` (e.g. if `pageSize: 25`, this would be `0` on page 1, `25` on page 2, etc.).
+
+##### `page.end`
+
+<p>
+**Type:** `number`
+</p>
+
+Index of last item on current page.
+
+##### `page.size`
+
+<p>
+**Type:** `number`
+</p>
+
+How many items per-page.
+
+##### `page.total`
+
+<p>
+**Type:** `number`
+</p>
+
+The total number of items across all pages.
+
+##### `page.currentPage`
+
+<p>
+**Type:** `number`
+</p>
+
+The current page number, starting with `1`.
+
+##### `page.lastPage`
+
+<p>
+**Type:** `number`
+</p>
+
+The total number of pages.
+
+##### `page.url.current`
+
+<p>
+**Type:** `string`
+</p>
+
+Get the URL of the current page (useful for canonical URLs)
+
+##### `page.url.prev`
+
+<p>
+**Type:** `string | undefined`
+</p>
+
+Get the URL of the previous page (will be `undefined` if on page 1).
+
+##### `page.url.next`
+
+<p>
+**Type:** `string | undefined`
+</p>
+
+Get the URL of the next page (will be `undefined` if no more pages).
+
+{/*
```

Comment:
You should probably remove this now!

Reply:
Thanks for catching this. It's been resolved in my new commit :)!